### ISSUE TITLE
It seems to reverse the schedules when running in some scenarios

### ISSUE THREAD

**Issue by HSchmale16** [aws-solutions/instance-scheduler-on-aws#411]
I have a schedule that seems to be miss-reported in the cli `describe-schedule-usage`. It should run all day every 3rd Tuesday of the month, and from 5am to 7pm the rest of the week. Yet the periods don't seem to be additive. Here's the cloud formation I used to create it.

```
  AmericaWeekdayScheduleDev:
    Type: 'Custom::ServiceInstanceSchedule'
    Properties:
      Name: US-DEV-BUSINESS-HOURS-WEEKDAYS-ONLY
      NoStackPrefix: 'True'
      Description: United States Business Hours on Mon-Fri, off on weekends
      ServiceToken: !Ref ServiceToken
      Enforced: 'True'
      Hibernate: 'True'
      Timezone: !Ref RegionTzAmerica
      Periods:
        - Description: Must run all day every 3rd Tuesday
          WeekDays: tue#3
          BeginTime: '00:00'
          EndTime: '05:00'
        - Description: Opening hours on weekdays
          BeginTime: '05:00'
          EndTime: '19:00'
          WeekDays: mon-fri
        - Description: Must run all day every 3rd Tuesday
          WeekDays: tue#3
          BeginTime: '19:00'
          EndTime: '23:59'
```

As it stands the output says it starts at 7pm and dies at midnight everyday. A schedule with just the middle period produces output as expected. Additionally the 3rd Tuesday runs from 5am to midnight instead of starting at midnight.

We are currently running version `(SO0030) - aws-instance-scheduler, version v1.3.2`

I've attached the output of `describe-schedule-usage` to this issue.
[results.txt](https://github.com/aws-solutions/aws-instance-scheduler/files/11055145/results.txt)

**Comment by CrypticCabub** [comment]
Hi @HSchmale16 

Is it just the describe-schedule-usage command that fails to report correctly or is the scheduler itself also behaving in the incorrect fashion described by the command?

**Comment by HSchmale16** [comment]
> Hi @HSchmale16 
> 
> Is it just the describe-schedule-usage command that fails to report correctly or is the scheduler itself also behaving in the incorrect fashion described by the command?

We're still in the prototyping phase for this so I'm testing the best I can with the cli. So I don't know if it's also affected in the actual scheduler.

**Comment by CrypticCabub** [comment]
Thanks!

I suspect the issue is with the describe command rather than the scheduler itself, but please let us know if you find out otherwise. For now I have filed this as a bug that needs further investigation with a particular focus on the CLI command behavior

**Comment by CrypticCabub** [comment]
Fixed in [v3.0.0](https://github.com/aws-solutions/instance-scheduler-on-aws/releases/tag/v3.0.0)
